src/plugins/content_management/common/rpc.ts

```diff
@@ -64,9 +64,85 @@ export interface CreateIn<
   options?: Options;
 }
 
+// -- Update content
+const updateSchemas: ProcedureSchemas = {
```

Comment:
Just following the older examples for now. It is clear that this types will change when the real RPC implementation will be delivered.